src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx

```diff
@@ -605,11 +651,128 @@ export const AutomatedAnalysisCard: React.FC<AutomatedAnalysisCardProps> = (prop
     );
   }, [isLoading, usingArchivedReport, usingCachedReport, reportStalenessTimer, reportStalenessTimerUnits]);
 
+  const filteredCategorizedLabels = React.useMemo(() => {
+    const filtered = filteredCategorizedEvaluation.filter(([_, evaluations]) => evaluations.length > 0);
+    if (filtered.length === 0) {
+      return (
+        <EmptyState>
+          <EmptyStateIcon icon={SearchIcon} />
+          <Title headingLevel="h4" size="lg">
+            {t(`AutomatedAnalysisCard.NO_RESULTS`)}
+          </Title>
+          <EmptyStateBody>{t('AutomatedAnalysisCard.NO_RESULTS_BODY')}</EmptyStateBody>
+          <EmptyStateSecondaryActions>
+            <Button variant="link" onClick={handleClearFilters}>
```

Comment:
How about the case where `Show N/A` is not checked and filtered results have N/A scores? Maybe add another clause for it?

![Screenshot from 2023-02-15 06-04-18](https://user-images.githubusercontent.com/68053619/219010424-b26532a3-c909-447a-88c5-0a46b29f2377.png)

Reply:
Are you saying I should change the wording of the text?